views/agreement.hbs

```diff
@@ -165,6 +267,11 @@ input[type="submit"] {
    all WHATWG <a href="https://whatwg.org/workstream-policy#workstream">Workstreams</a>, current and future
   </label>
 
+  <label>
+   <input type="radio" name="scope" value="invited">
+   I'm invited by the <a href="https://whatwg.org/sg-agreement#steering-group">Steering Group</a> and want to join all the <a href="https://whatwg.org/workstream-policy#workstream">Workstreams</a> I have been invited to
```

Comment:
This puts more work on the verifier, as they will have to go find the invitation and update the database manually with the workstreams.

It would be ideal if, instead, the participant filling out the form selected the workstreams (like any other individual), and then the verifier just checked that the filled-out workstreams match the invited-to workstreams.

(BTW, both of these workflows assume the invitations are posted somewhere the verifier has access to; I haven't checked to see if that's the case.)

Reply:
The idea is that we put the result of the invite in JSON as well and the bot joins the two JSON resources when determining the status.

Reply:
I guess that works. The SG would hand-maintain the JSON file?

Reply:
Yeah, it's expected to be a rare process so that seems fine. To be sure we could require a PR for changes to that file in particular and create a checker. (As it's a bit more involved than removing a record or flipping a boolean.)